### PR TITLE
style(searchform): restore outline on search form buttons

### PR DESCRIPTION
**Pull request by cweeii**
image

image

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @cweeii has signed the CLA. Thanks for the pull request!

File: src/components/search-form/SearchForm.scss [hunk 1]
```diff
@@ -17,11 +17,6 @@
         padding: 0;
         width: 30px;
 
-        &:hover,
```
Comment (jstoffan) [inline src/components/search-form/SearchForm.scss]: This is also used in ContentExplorer and MetadataSidebar. Can you check those quickly to make sure they don't have any glaring issues after this change?

Comment (cweeii) [reply src/components/search-form/SearchForm.scss]: Just took a look at both and no glaring issues.  Only difference is the outline shows up now when you tab to it.

**Comment by ConradJChan** [comment]
Was there previously a different hover/focus effect in place of outline?

**Comment by patrickpaul** [comment]
Given recent changes to the design spec, this PR is no longer applicable.

**Pull request closed without merging**